docker/1.14/Dockerfile.eia

```diff
@@ -0,0 +1,86 @@
+FROM ubuntu:16.04
+LABEL com.amazonaws.sagemaker.capabilities.accept-bind-to-port=true
+
+ARG TFS_SHORT_VERSION=1.14
+ARG TFS_VERSION=1-14-0
+ARG TFS_EI_VERSION=1-2
+ARG PYTHON=python3
+ARG PIP=pip3
```

Comment:
pip should not be an argument here. It should be determined by the python version

Reply:
Removed PIP as an argument.